### ISSUE TITLE
Discussion about how to best go from Rnw to HTML.

### ISSUE THREAD

**Issue by seandavi** [seandavi/F1000R_BiocWorkflows#1]

**Comment by seandavi** [comment]
Pandoc can go from latex to markdown and then html.

http://pandoc.org/demos.html

**Comment by kasperdanielhansen** [comment]
I think we should instead try to standardize on Rmarkdown, or at least `knitr`. .  Most documents written in Rnw format can be converted easily.  There is `knitr::Sweave2knitr()` for going from Sweave syntax tex+R to knitr syntax tex+R.

**Comment by seandavi** [comment]
I agree. I have been pretty successful converting several old-style vignettes without a lot of work.

**Comment by grimbough** [comment]
I've commited a script to the folder 'processRnw' which does a reasonable job of converting the knitr Rnw in Gordon Smyth's workflow into an Rmd.  This uses a mix of regex to change some tags manually, and then pandoc.  It's not perfect, but I guess it depends on what we're aiming for.

If we just want to give someone a head start before doing some manual work to produce a markdown vignette, which they will then submit to the workflow svn/builder then it's probably fine.

Alternatively, if we'd actually like people to be able to submit a knitr Rnw vignette, which is parsed by the workflow builder and displayed as html on the Bioc webpages, then I think something more comprehensive is required.

**Comment by kasperdanielhansen** [comment]
I'll take a look.  We have two different goals
- (short term) make the already submitted workflows into something suitable
  for the website
- identify a process where an author can make a workflow which is both
  submitted to F1000 and to the website.

For the second goal, if we need to be draconian in terms of file formats
(you can only use Rmd + ....) then so be it.  I don't think we can make
this process easy without standardizing on some input format (of course, it
would be great if we could).

Kasper

**Comment by seandavi** [comment]
Totally agree with Kasper here.  Just settling on a standard will allow us to describe the process in detail.  From there, we can always extend and modify.  In the sense of this issue, we could simply offer advice of how to go from Rnw to Rmd, but that will be left as an exercise for the author.  We simply take Rmd.

**Comment by seandavi** [comment]
That said, it looks like 145f8d9a6 makes some headway on a script from Rnw to Rmd.
